packages/playwright/src/reporter.ts

```diff
@@ -85,13 +86,17 @@ export default class ReplayPlaywrightReporter implements Reporter {
   constructor(config: ReplayPlaywrightConfig) {
     setUserAgent(`${packageName}/${packageVersion}`);
 
+    const accessToken = getAccessToken(config);
+
     initLogger(packageName, packageVersion);
     mixpanelAPI.initialize({
-      accessToken: getAccessToken(config),
+      accessToken,
       packageName,
       packageVersion,
     });
-
+    if (accessToken) {
+      initLaunchDarklyFromAccessToken(accessToken);
```

Comment:
This is async but we can't even await things here, this is race'y. But what should be done about it - should `getFeatureFlagValue` await the pending identification?

Reply:
Hmm... `initLaunchDarklyFromAccessToken` is async because it fetches auth info.

The reporter base class (in `test-utils`) already awaits auth info in order to initialize the Grafana logger. Should we move LD initialization in there?

>  should `getFeatureFlagValue` await the pending identification?

It already does :) That's how I designed it.

Reply:
> Hmm... initLaunchDarklyFromAccessToken is async because it fetches auth info.

The same is true about MixPanel's initialization - both should get the same treatment here. 

> The reporter base class (in test-utils) already awaits auth info in order to initialize the Grafana logger. Should we move LD initialization in there?

I'll look into this.

> It already does :) That's how I designed it.

It waits on the LD's initialization - but is this the same as the user's identification? 🤔

Reply:
Perhaps I misremembered what it's doing 😆 You're right. I think waiting some small amount (before falling back to the default user profiles) seems reasonable?